project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 resolvers += Resolver.sonatypeRepo("releases")
 addSbtPlugin("com.47deg"          % "sbt-org-policies"         % "0.11.3")
-addSbtPlugin("com.47deg"          % "sbt-microsites"           % "0.9.0")
+addSbtPlugin("com.47deg"          % "sbt-microsites"           % "0.9.3-SNAPSHOT")
```

Comment:
Snapshot are not releases.

If that snapshot include a needed feature for this PR we better wait to have it as part of a release.

Reply:
That is a mistake, I´ll set the `0.9.2` wich is the latest. There is not any needed feature in the SNAPSHOT version